### PR TITLE
Windows test accomodations

### PR DESCRIPTION
**Pull request by compnerd**
Replace this paragraph with a description of your changes and rationale. Provide links to external references/discussions if appropriate.

Resolves [SR-NNNN](https://bugs.swift.org/browse/SR-NNNN).

### CODE REVIEW

**Comment by compnerd** [comment]
CC: @moiseev

**Comment by compnerd** [comment]
@swift-ci please test

**Comment by compnerd** [comment]
@swift-ci please test Linux platform

**Comment by compnerd** [comment]
@swift-ci please test Linux platform

**Review by lorentey — APPROVED** [review]
👍
(Blushing for not realizing those slashes will be an issue on Windows...)

File: stdlib/private/StdlibUnittest/StdlibUnittest.swift [hunk 1]
```diff
@@ -928,7 +928,10 @@ class _ParentProcess {
     let (_, stderrThread) = _stdlib_thread_create_block({
       while !self._childStderr.isEOF {
         self._childStderr.read()
-        while let line = self._childStderr.getline() {
+        while var line = self._childStderr.getline() {
+          if let cr = line.firstIndex(of: "\r") {
+            line.remove(at: cr)
```
[the remark this answers is not in the document]
Comment (compnerd) [reply stdlib/private/StdlibUnittest/StdlibUnittest.swift]: Yeah, that would be great to fix, I just have not been able to do that properly for some reason.

**Pull request merged**